TelepresenceBot/server-unit/test/test.js

```diff
@@ -0,0 +1,34 @@
+var chai = require('chai'),
+    mocha = require('mocha'),
+    should = chai.should();
+
+var io = require('socket.io-client');
+
+var server, options = {
+    transports: ['websocket'],
+    'force new connection': true
+};
+
+describe("echo", function () {
+
+    beforeEach(function (done) {
+        server = require('../core/testServer').server;
+        done();
+    });
+
+    it("echos message", function (done) {
```

Comment:
Tests are not complete atm, I did manual testing just to get this off of the ground. Half the problem with this stuff is I need to figure out how to test it 😆  As per the description, I plan on doing the connecting with `socketIO` and testing in the next PR.